annif/project.py

```diff
@@ -42,6 +43,7 @@ class AnnifProject(DatadirMixin):
 
     # default values for configuration settings
     DEFAULT_ACCESS = "public"
+    DOC_BATCH_SIZE = 32
```

Comment:
This constant is declared here in AnnifProject, but its only uses are to pass it as a parameter to corpus.doc_batches() and 2 out of 3 of these are in cli.py. Would it be better to move this constant to the DocumentCorpus class and simply use the constant value within doc_batches instead of passing it around as a parameter?

The alternative would be to make this configurable per project (just like vocab, language etc.), but I'm not sure if that would add any value.